ionoscloud/resource_dbaas_mariadb_cluster.go

```diff
@@ -144,6 +144,24 @@ func resourceDBaaSMariaDBCluster() *schema.Resource {
 					},
 				},
 			},
+			"backup": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Description: "Properties configuring the backup of the cluster. Immutable, change forces re-creation of the cluster.",
+				Optional:    true,
```

Comment:
@rmocanu-ionos if `backup` is not specified, is there a default location in which the backups will be created? Asking to see if this attribute also needs to be marked as `Computed: true` or no.

Reply:
Yes, the default location is DEFAULT